postgres/tests/test_statements.py

```diff
@@ -393,13 +417,17 @@ def test_failed_explain_handling(
     expected_error_tag,
     explain_function_override,
     expected_fail_count,
+    skip_on_versions,
 ):
     dbname = "datadog_test"
     if explain_function_override:
         dbm_instance['query_samples']['explain_function'] = explain_function_override
     check = integration_check(dbm_instance)
     check._connect()
 
+    if skip_on_versions is not None and check.version.major in skip_on_versions:
+        return True
```

Comment:
Better to use `pytest.skip("reason")` because then it will clearly show up as a skipped test in the test results. `return True` will mark it a success. 

Also better to use `POSTGRES_VERSION` to check which version it is instead of running the check. It's imported at the top of this file `from .common import DB_NAME, HOST, PORT, POSTGRES_VERSION`. 

That way we can skip without having to actually run a part of the check which is querying the DB.

Reply:
👍

Reply:
Using `POSTGRES_VERSION` doesn't seem to work as it's expecting all version even up to 14 so it would mean that first test for datatypemismatch will fail because it doesn't include 14 in the `skip_on_versions` . 
print statement below for variables: 
`skip_on_versions: [13, 12, 11, 10, 9] POSTGRES_VERSION: 14 check.version.major: 13`